docs/GettingStarted.md

```diff
@@ -707,6 +709,54 @@ Where `options` is an optional `Hash` that accepts the following parameters:
 | ``service_name`` | Service name used for `resque` instrumentation | resque |
 | ``workers`` | An array including all worker classes you want to trace (eg ``[MyJob]``) | ``[]`` |
 
+### Sequel
+
+The Sequel integration traces queries made to your database.
+
+```ruby
+require 'sequel'
+require 'ddtrace'
+
+# Connect to database
+database = Sequel.sqlite
+
+# Create a table
+database.create_table :articles do
+  primary_key :id
+  String :name
+end
+
+Datadog.configure do |c|
+  c.use :sequel, options
+end
+
+# Perform a query
+articles = database[:articles]
+articles.all
+```
+
+Where `options` is an optional `Hash` that accepts the following parameters:
+
+| Key | Description | Default |
+| --- | --- | --- |
+| ``service_name`` | Service name used for `sequel.query` spans. | Name of database adapter (e.g. `mysql2`) |
+| ``tracer`` | A ``Datadog::Tracer`` instance used to instrument the application. Usually you don't need to set that. | ``Datadog.tracer`` |
+
+Only Ruby 2.0+ is supported.
+
+**Configuring databases to use different service names**
```

Comment:
`to use different settings` -> mostly to explain that this per-client configuration is generic and works for any `options` we'll add in the future.

Reply:
Make sense.